libraries/radspreadstreamprocessing/overview.md

```diff
@@ -12,13 +12,40 @@ position: 0
 
 Thank you for choosing Telerik **RadSpreadStreamProcessing**!
 
-**RadSpreadStreamProcessing** is a library that allows you create big spreadsheet documents and export them to the most common file formats with a great performance and minimal resources.
+This article briefly explains the Spread Stream library specifics as what is spread streaming, how it works compared to the **RadSpreadProcessing** library and when to use it.
```

Comment:
This article briefly explains the specifics of RadSpreadStreamProcessing - what is spread streaming, how it works compared to the **RadSpreadProcessing** library and when to use it.